pkg/gallery/models.go

```diff
@@ -183,54 +179,3 @@ func InstallModel(basePath, nameOverride string, config *Config, configOverrides
 
 	return nil
 }
-
-type progressWriter struct {
```

Comment:
trying to understand where you moved the `progressWriter`, the diff is quite huge for Github hand hard to follow up

Reply:
processWriter got moved down into pkg/utils/uri.go since unless I made a merging error somewhere along the way, it's only used within uri.go's `DownloadFile` function